src/llvm.cpp

```diff
@@ -1852,4 +1896,8 @@ void printIR(llvm::Type *value) {
     puts("\n");
 }
 
+void printModule(llvm::Module *module) {
```

Comment:
https://github.com/kai-language/kai-c/pull/49/files#diff-cf87bdfd88530e0c5c46f9b95cf006beR1885
There is already and overload for this called printIR let's just use it for now